core/src/main/java/org/wildfly/extras/sunstone/api/impl/ec2/EC2CloudProvider.java

```diff
@@ -73,6 +74,22 @@ protected JCloudsNode createNodeInternal(String name, Map<String, String> overri
         return new EC2Node(this, name, overrides);
     }
 
+    @Override
+    protected String postProcessNodeGroupWhenCreatingNode(String nodeGroup) {
+        // JClouds place a limitation of 63 characters on the node group (see https://github.com/wildfly-extras/sunstone/issues/27)
```

Comment:
If this is a JClouds limitation, then we should enforce it for all cloud providers?